### PR TITLE
added friends faker and tests

### PR DESCRIPTION
**Pull request by JIntrocaso**
Added Friends faker data and tests following structure and standards observed in project.

### CODE REVIEW

**Comment by JIntrocaso** [comment]
@mrstebo I submitted those changes requested from your review. Thanks for that tip about the extra data.

File: src/FakerDotNet/Faker.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@ public static class Faker
         public static INumberFaker Number { get; } = Container.Number;
         public static IRandomFaker Random { get; } = Container.Random;
         public static ITimeFaker Time { get; } = Container.Time;
+        public static IFriendsFaker Friends { get; } = Container.Friends;
```
Comment (mrstebo) [inline src/FakerDotNet/Faker.cs]: These properties are supposed to be in alphabetical order. It makes it easier for new contributors to know where to add the new properties into the `Faker` and `FakerContainer` classes :+1:

File: tests/FakerDotNet.Tests/FakerContainerTests.cs [hunk 2]
```diff
@@ -80,5 +80,11 @@ public void Zelda_returns_IZeldaFaker()
         {
             Assert.IsInstanceOf<IZeldaFaker>(_fakerContainer.Zelda);
         }
+
+        [Test]
+        public void Friends_returns_IFriendsFaker()
```
Comment (mrstebo) [inline tests/FakerDotNet.Tests/FakerContainerTests.cs]: Same as above. These tests need to be in the same position as the properties on the class (in alphabetical order).

File: tests/FakerDotNet.Tests/FakerTests.cs [hunk 3]
```diff
@@ -72,5 +72,11 @@ public void Zelda_returns_IZeldaFaker()
         {
             Assert.IsInstanceOf<IZeldaFaker>(Faker.Zelda);
         }
+
+        [Test]
+        public void Friends_returns_IFriendsFaker()
```
Comment (mrstebo) [inline tests/FakerDotNet.Tests/FakerTests.cs]: Same as above :point_up:

**Comment by mrstebo** [comment]
Thanks for the contribution @JIntrocaso :tada:

**Comment by JIntrocaso** [comment]
Thanks for walking me through it! Might try another to see if I can do it right by myself. 😆

**Pull request merged**